src/derivatives/common-data-types.md

```diff
@@ -254,6 +254,77 @@ be propagated to the JSON description of the derivative unless the processing
 makes them invalid (for example, if a source 4D image is averaged to create a single
 static volume, a `RepetitionTime` property would no longer be relevant).
 
+## descriptions.tsv
+
+To keep a record of processing steps applied to the data, a `descriptions.tsv` file MAY be used.
+The `descriptions.tsv` file MUST contain at least the following two columns:
+
+-   `label`
+-   `description`
+
+This file MAY be located at the root of the derivative dataset,
+or at the subject or session level
+([Inheritance Principle](../common-principles.md#the-inheritance-principle)).
+
+The `label` column contains the labels used with the [`desc entity`](../appendices/entities.md#desc),
+within the particular nesting that the `description.tsv` file is placed.
+For example, if the `descriptions.tsv` file is placed at the root of the derivative dataset,
+its `label` column SHOULD contain all labels of the [`desc entity`](../appendices/entities.md#desc)
+used across the entire derivative dataset.
+
+The `description` column contains human-readable descriptions of the processing steps.
+
+The use of `description.tsv` files together with the [`desc entity`](../appendices/entities.md#desc)
```

Comment:
it should be plural `descriptions.tsv`

Reply:
fixed in https://github.com/bids-standard/bids-specification/pull/1613/commits/5ab8b1e34862f8a2fc9e8cc4ac5ae754c8b11e53